engine/executor/server/mocks/metastore_mock.go

```diff
@@ -9,8 +9,7 @@ import (
 	reflect "reflect"
 
 	gomock "github.com/golang/mock/gomock"
-	extension "github.com/pingcap/tiflow/engine/pkg/meta/extension"
-	metaclient "github.com/pingcap/tiflow/engine/pkg/meta/metaclient"
+	common "github.com/pingcap/tiflow/engine/pkg/meta/model"
```

Comment:
Please unify all the import alias.

Reply:
This file is autogenerated by `mockgen`, so the alias here may be different  from other cases.

Reply:
Fix it as `model "github.com/pingcap/tiflow/engine/pkg/meta/model"`